app/src/main/java/com/sun/android/MainActivity.kt

```diff
@@ -1,11 +1,39 @@
 package com.sun.android
 
+import android.content.Intent
 import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        var tvTitle = findViewById(R.id.tvTitle) as TextView
```

Comment:
//các biến này chỉ gán 1 lần thì dùng val
nên
```suggestion
        //var textViewTitle : TextView = findViewById(R.id.tvTitle)
        //var textViewTitle : TextView = findViewById<TextView>(R.id.tvTitle)
```